src/main/resources/languages/en/messages.yml

```diff
@@ -139,7 +139,7 @@ guide:
       translator: '&9Translator'
 
 messages:
-  not-researched: '&4You do not have enough knowledge to understand this'
+  not-researched: '&4You do not have enough knowledge to understand this. You need to research %item%'
```

Comment:
```suggestion
  not-researched: '&4You do not have enough knowledge to understand this. &cYou will need to unlock &7"%item%&7"'
```

Reply:
Do we maybe want to reset before %item% just so if the item has no colour code it is white?

Reply:
It will be gray, that's why I added &7 there

Reply:
exactly why I'm saying do we want to reset. So quotes are gray but item is true color no matter what (even if white)

Reply:
You mean reset as in `ChatColor.stripColor` and not `ChatColor.RESET` then?